plugins/com.google.cloud.tools.eclipse.appengine.facets/src/com/google/cloud/tools/eclipse/appengine/facets/FlexFacetInstallDelegate.java

```diff
@@ -47,15 +48,22 @@ private void createConfigFiles(IProject project, IProgressMonitor monitor) throw
     FlexDeployPreferences flexDeployPreferences = new FlexDeployPreferences(project);
     String appYamlParentPath = flexDeployPreferences.getAppEngineDirectory();
     IFolder appYamlParentFolder = project.getFolder(appYamlParentPath);
-    IFile appYaml = appYamlParentFolder.getFile(AppEngineTemplateUtility.APP_YAML);
+    IFile appYaml = appYamlParentFolder.getFile("app.yaml");
     if (appYaml.exists()) {
       return;
     }
+    
+    if (!appYamlParentFolder.exists()) {
+      appYamlParentFolder.create(true, true, subMonitor.newChild(5));
```

Comment:
This works only when the parent of `appYamlParentFolder` exists, so probably better to use `ResourceUtils.createFolders()`.